manifests/rgw.pp

```diff
@@ -139,7 +139,8 @@
   }
 
   package { $pkg_radosgw:
-    ensure => installed,
```

Comment:
Please note that this part is hardcoded, so this could not be fed from e.g. cccp as a class parameter